apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -1789,19 +1788,15 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
       {processor, _} = Core.new_piggybacks(processor, [%{tx_hash: tx_hash, output_index: 1}])
       {processor, _} = Core.new_piggybacks(processor, [%{tx_hash: tx_hash, output_index: 2}])
 
-      finalization1 = %{in_flight_exit_id: ife_id, output_index: 1}
-
-      {:ok, processor, [{:put, :in_flight_exit_info, {_, exit_info}}]} =
-        Core.finalize_in_flight_exits(processor, [finalization1])
+      {:ok, processor, [{:put, :in_flight_exit_info, _}]} =
+        Core.finalize_in_flight_exits(processor, [%{in_flight_exit_id: ife_id, output_index: 1}])
 
-      assert expect_finalized_outputs(exit_info, [1], [2])
-
-      finalization2 = %{in_flight_exit_id: ife_id, output_index: 2}
+      [_] = Core.get_active_in_flight_exits(processor)
 
-      {:ok, _, [{:put, :in_flight_exit_info, {_, exit_info}}]} =
-        Core.finalize_in_flight_exits(processor, [finalization2])
+      {:ok, processor, [{:put, :in_flight_exit_info, _}]} =
+        Core.finalize_in_flight_exits(processor, [%{in_flight_exit_id: ife_id, output_index: 2}])
 
-      assert expect_finalized_outputs(exit_info, [1, 2], [])
+      assert [] == Core.get_active_in_flight_exits(processor)
```

Comment:
ah I only noticed this now, `get_active_in_flight_exits` and `get_in_flight_exits` - it seems like only one of these should be `def` and tested here. I think `get_in_flight_exits` should be removed and all the `CoreTest` should be adapted to test the other. `get_active_in_flight_exits` seems to be the only one used in the `ExitProcessor`